### ISSUE TITLE
Fix accessibility violations

### ISSUE THREAD

**Issue by PRINCESANCHEZ** [elyra-ai/canvas#2218]
Ref: 16314

<img width="866" alt="Screenshot 2024-10-21 at 10 22 45 AM" src="https://github.com/user-attachments/assets/500d59d2-5a30-4e11-9fa0-10ba4ce540de">
<img width="861" alt="Screenshot 2024-10-21 at 9 58 29 AM" src="https://github.com/user-attachments/assets/87d98a5e-e123-4238-acb5-1c19059a830f">
